apps/backlite/boot.js

```diff
@@ -11,26 +11,36 @@
   //Set LCD to zero every reboot
   let s = require("Storage").readJSON("setting.json", 1) || {};
   s.brightness = 0;
+  if (!("lcdTimeout" in s)) s.lcdTimeout = 5; // fallback so  logic doesn't break
   require("Storage").writeJSON("setting.json", s);
   //remove large settings object from memory
-  delete s;
+  s=null;
   const longPressTime=400; //(ms)
-
+  var longPressTimer;
   Bangle.on('lock', function(isLocked) {
     Bangle.setLCDBrightness(0);
 
     if (!isLocked) {
       // Just unlocked — give a short delay and check if BTN1 is still pressed
-      setTimeout(() => {
+      longPressTimer=setTimeout(() => {
         if (digitalRead(BTN1)) {
           //set brightness until. locked.
           Bangle.setLCDBrightness(getSettings().brightness);
         } else {
           Bangle.setLCDBrightness(0);
         }
+        clearTimeout(longPressTimer)
```

Comment:
Here there is no timeout to clear, since it's already fired. You could set it to `null` (not deleting here, since we want the variable to remain for future sets)

```suggestion
        longPressTimer = null;
```